### ISSUE TITLE
How can I get current rule id in current field props

### ISSUE THREAD

**Issue by developer-anilpatidar** [react-querybuilder/react-querybuilder#116]
I need to hide field selector according to operators but I didn't get rule id in field selector props. How can I get current rule id in current field props?

**Comment by jakeboone02** [comment]
You want to hide the fields selector when certain operators are chosen?  Seems strange...how would the user know what field they were working with?  Anyway, if you had the `id` as a prop, wouldn't you have to recursively search through the query to find the rule to get the operator?  Wouldn't it be easier to just get the `operator` as a prop directly?  Provide some sample code for how you'd like it to work and we'll see if we can accommodate your use case.

**Comment by MayankSG** [comment]
I don't have `id` in props, I have field props like that: 

```
className: "rule-fields _1jihlcN7mKfwxFNa1rjBUt"
handleOnChange: ƒ (value)
level: 0
options: (328) [{…}, {…} …]
title: "Fields"
value: "width"
```

My scenario is like that I am using function like add(arg1, arg2) in the operator list when I select function operator then I need to hide  fields and value, I am checking field value in rules and find out the rule but it's only working when I have only one rule, Many rules cases it's not working because field value will be same for multiple rules.

It would be good if I can receive `operator` or `rule id` in field props, just like a valueEditor.
 In the valueEditor props, I am getting operator vale like that:

```
className: "rule-value _1jihlcN7mKfwxFNa1rjBUt"
field: "height"
handleOnChange: ƒ (value)
level: 0
operator: "ipv4_is_private"       // can i get this operator value in field
title: "Value"
value: "height"
```

**Comment by developer-anilpatidar** [comment]
I don't have `id` in props, I have field props like that: 

```
className: "rule-fields _1jihlcN7mKfwxFNa1rjBUt"
handleOnChange: ƒ (value)
level: 0
options: (328) [{…}, {…} …]
title: "Fields"
value: "width"
```

My scenario is like that I am using function like add(arg1, arg2) in the operator list when I select function operator then I need to hide  fields and value, I am checking field value in rules and find out the rule but it's only working when I have only one rule, Many rules cases it's not working because field value will be same for multiple rules.

It would be good if I can receive `operator` or `rule id` in field props, just like a valueEditor.
 In the valueEditor props, I am getting operator vale like that:

```
className: "rule-value _1jihlcN7mKfwxFNa1rjBUt"
field: "height"
handleOnChange: ƒ (value)
level: 0
operator: "ipv4_is_private"       // can i get this operator value in field
title: "Value"
value: "height"
```

**Comment by jakeboone02** [comment]
Yes, I think we can add `operator` as a prop to the `fieldSelector` component.

**Comment by jakeboone02** [comment]
Fixed by 32c7f4669.  Will be released soon.
